rules/graphql-when-to-use/rule.md

```diff
@@ -38,61 +36,53 @@ GraphQL differs from REST in that REST is concerned with  **resources** and Grap
       https://graphql.org/learn/queries/#mutations
 * **Schema:** Your GraphQL schema is the complete map of data your GraphQL server exposes. This defines all the data that your clients can request, and all the changes they can make. Specific queries or mutations are not explicitly defined; rather, you specify their structure. Just like with a SQL database, you write whatever queries you want against it, and if the schema supports them, they will run successfully.
 
-
 ### Advantages of GraphQL
 
-
 * **No under-fetching:** Sometimes a REST resource may only give you part of what you need. For example, if you need to know all the orders placed by a customer, what date they were ordered, and their current status, but the initial customer search resource only returns the details of the customer, this is called under-fetching. You then need to use the customer’s ID to query your orders resource for all orders matching that customer ID.
 
-
-
-::: bad  
-![Figure: Bad Example – REST API does not return order information with a customer query, so a second API call is needed](graphql-bad-example-underfetching.png)  
+::: bad\
```

Comment:
Hey @ozairashfaqueSSW, 
Please remove the trailing slashes `\` (it's a bug with the Netlify CMS editor)